full-stack-tests/backend/src/integration/CloudSqlite.test.ts

```diff
@@ -128,6 +129,135 @@ describe("CloudSqlite", () => {
     await CloudSqliteTest.uploadFile(testContainers[2], caches[0], "testBim", testBimFileName);
   });
 
+  it("should query bcvHttpLog", async () => {
```

Comment:
@nick4598, let's just leave this test out of this PR, because i've made substantial changes to this stuff on [this PR](https://github.com/iTwin/itwinjs-core/pull/5455) and all this will cause lots of conflicts. After the next native build, we can combine this test with the above tests.